src/theme/SearchBar/lib/DocSearch.js

```diff
@@ -0,0 +1,276 @@
+import Hogan from 'hogan.js';
```

Comment:
Where is this coming from?  I don't see it in this PR

Reply:
Hogan is a dependency comes with `docusaurus-lunr-search`, I don't know it is not installed automatically for some reason.